src/relayclient/GsnTest.ts

```diff
@@ -0,0 +1,57 @@
+import Web3 from 'web3'
+
+import CommandsLogic, { DeploymentResult } from '../cli/CommandsLogic'
+import { GsnDevProvider } from './GsnDevProvider'
+import { configureGSN } from './GSNConfigurator'
+import { DevGSNConfig } from './DevRelayClient'
+import { networks } from '../cli/utils'
+
+interface TestEnvironment {
+  deployment: DeploymentResult
+  devProvider: GsnDevProvider
+}
+
+class GsnTestClass {
+  private fixture?: TestEnvironment
+
+  /**
+   *
+   * @param host
+   * @param paymaster TODO: will allow using custom paymaster (need to provide ABI file contents)
+   * @return
+   */
+  async start (host?: string, paymaster?: any): Promise<void> {
+    const _host: string = host ?? networks.get('localhost') ?? ''
+    const commandsLogic = new CommandsLogic(_host, configureGSN({}))
+    const from = await commandsLogic.findWealthyAccount()
+    if (from == null) {
```

Comment:
findWealthyAccount already throws. redundant check.

Reply:
Made 'findWealthyAccount' not throw. returning undefined is much cleaner, type-safe API imo